Core/HLE/sceKernel.cpp

```diff
@@ -276,6 +276,7 @@ void __KernelDoState(PointerWrap &p)
 		__VideoPmpDoState(p);
 		__AACDoState(p);
 		__UsbGpsDoState(p);
+		// __UsbCamDoState(p); // not yet ;D
```

Comment:
Not yet, why? Either explain in the comment, or remove the line entirely please.

Reply:
I find challenging to maintain backward compatibility for save states and I doubt that the current version contain enough information to cover every scenario. I'll remove it